passes/cmds/bugpoint.cc

```diff
@@ -304,6 +319,9 @@ struct BugpointPass : public Pass {
 		bool fast = false, clean = false;
 		bool modules = false, ports = false, cells = false, connections = false, assigns = false, updates = false, has_part = false;
 
+		log_header(design, "Executing BUGPOINT pass (minimize testcases).\n");
```

Comment:
I vaguely recall intentionally not adding a header for some reason. But if you think a header looks OK, let's keep it.